core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -281,6 +297,7 @@ static function (array $value): array {
                     ->end()
                 ->end()
                 ->variableNode('localconfig')
+                    ->info('Allows to set TL_CONFIG variables. Note that any property set here will override the localconfig.php file, so changing them in the Contao back end will not have any effect.')
```

Comment:
```suggestion
                    ->info('Allows to set TL_CONFIG variables, overriding settings stored in localconfig.php. Changes in the Contao back end will not have any effect.')
```
@leofeyer @xchs wdyt?

Reply:
Fine with me. 👍